tests/expectations/compiler/compiler/array_without_size/length.leo.out

```diff
@@ -17,6 +17,6 @@ outputs:
               type: bool
               value: "true"
     initial_ast: d8e3de13a7f12866871e4117b7c4cf9872350154b19e82087fe8100adeba1354
-    imports_resolved_ast: d8e3de13a7f12866871e4117b7c4cf9872350154b19e82087fe8100adeba1354
-    canonicalized_ast: d8e3de13a7f12866871e4117b7c4cf9872350154b19e82087fe8100adeba1354
-    type_inferenced_ast: 06be9e7e193c4fcdca75ba8218de79dcc8317b1eb63d9c2f33ca790c614a472b
+    imports_resolved_ast: 37de52ab186133dccfc290bed3f417685ce9ee5de37c820e0e8e3d2a17804bcc
```

Comment:
How could these expectations change? 🤨

Reply:
I'm not exactly sure but PRs #1347 and #1348 seem to be failing from the same tests. 
This is interesting especially in the case of #1347, which I would've expected to pass since it is just a version bump.

Reply:
@d0cd thanks for pointing out. Somehow we missed it, and these two got the wrong expectations on master. 😕